client/app/components/login/login.html

```diff
@@ -3,24 +3,25 @@
   <div flex="nogrow" layout="column">
     <div flex id="loginContent" layout-padding class="md-whiteframe-2dp">
       <form login-form ng-submit="$ctrl.login()">
-        <h2>Login</h2>
+        <h2 translate>LOGIN.TITLE</h2>
```

Comment:
Personally I prefer the style where you keep the translate key as the attribute value, (I think `translate="LOGIN.TITLE"`) so there is no chance the user sees the raw key rendered as text.

and does it HAVE TO SHOUT?